ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/AbstractCallOperation.java

```diff
@@ -36,6 +40,43 @@
       new OperationResult(
           Optional.empty(), Optional.of(ExceptionalHaltReason.INSUFFICIENT_STACK_ITEMS));
 
+  public static class CodeLoader extends CacheLoader<Account, Code> {
+    @Override
+    public Code load(final Account key) throws Exception {
+      if (!key.hasCode()) {
+        throw new IllegalArgumentException("account " + key + " has no executable code");
```

Comment:
this needs a test. This feels like a good way to harm Besu, one can craft an invalid transaction that makes it throw exceptions.